src/components/Greetings.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRandomGreeting } from '../redux/greetings/main';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRandomGreeting());
+  }, []);
+
+  // const greeting = useSelector((state) => state.greetings.greeting);
+  // console.log(greeting, 'text');
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

Reply:
Thank you